Doc/c-api/arg.rst

```diff
@@ -419,12 +419,24 @@ API Functions
    Returns true on success; on failure, it returns false and raises the
    appropriate exception.
 
+   .. note::
+
+      The *keywords* parameter declaration is :c:expr:`char * const *` in C and
+      :c:expr:`const char * const *` in C++.
+      This can be overridden by defining the macro :c:macro:`PY_CXX_CONST`
+      before including :file:`Python.h` as ``const`` for the latter and as
+      empty value for the former.
```

Comment:
```suggestion
      This can be overridden with the :c:macro:`PY_CXX_CONST` macro.
```

Refer to the newly-added macro's documentation instead of needing to repeat these details here.